### DISCUSSION TITLE
Max The Curse Of The Brotherhood

### DISCUSSION THREAD

**Discussion by sk03032001** [skyline-emu/skyline#967] [Game Requests]
When will work Max The Curse Of The Brotherhood
In skyline emulator?

**Comment by belmont31** [comment]
hello the game is playable with good fps but no audio ingame with build 1085 ! you have to use a save state to get pass the intro

for complet video you can chek : Zombla Med   : chanel on youtube 

![max](https://user-images.githubusercontent.com/104656222/167652366-2f3f75b6-24af-4723-8f1a-85d6f076de53.png)
